queries/rust/highlights.scm

```diff
@@ -9,7 +9,7 @@
   name: (identifier) @constant)
 ; Assume all-caps names are constants
 ((identifier) @constant
- (#match? @constant "^[A-Z][A-Z\\d_]+$'"))
+ (#match? @constant "^[A-Z][A-Z\\d_]*$"))
```

Comment:
this had an extra `'` at the end somehow, and one letter vars were being excluded.